src/js/App/Header/AppFilter.js

```diff
@@ -66,43 +67,53 @@ const AppFilterDropdown = ({ isLoaded, setIsOpen, isOpen, filterValue, setFilter
     isOpen={isOpen}
     ouiaId="App Filter"
   >
-    <div className="content">
-      {isLoaded ? (
-        <React.Fragment>
-          <Flex className="search">
-            <SearchInput
-              data-ouia-component-id="app-filter-search"
-              placeholder="Find application or service"
-              value={filterValue}
-              onChange={(val) => setFilterValue(val)}
-              onClear={() => setFilterValue('')}
-            />
-          </Flex>
-          {filteredApps?.length > 0 ? (
-            <div className="gallery">
-              {filteredApps.map((app) => (
-                <App key={app.id} {...app} />
-              ))}
+    {ReactDOM.createPortal(
+      <>
```

Comment:
I don't think you have to use Fragment here. But if that's true please use the full `React.Fragment` component. I tent to overlook these `<>`.